DataPipeline-Demo-Replace-Log4j-with-SLF4J-in-Gradle/src/main/java/com/northconcepts/datapipeline/ReplaceLog4jWithSlf4jInGradle.java

```diff
@@ -1,3 +1,9 @@
+/*
+ * Copyright (c) 2006-2021 North Concepts Inc.  All rights reserved.
+ * Proprietary and Confidential.  Use is subject to license terms.
+ * 
+ * https://northconcepts.com/data-pipeline/licensing/
```

Comment:
Actually, the demos are licensed under Apache License 2.0.

```suggestion
 * Copyright (c) 2006-2021 North Concepts Inc.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 * 
 *     http://www.apache.org/licenses/LICENSE-2.0
 * 
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 * 
```

Reply:
@DeleTaylor Done.
Sorry, I wasn't aware that demo should use apache license 2.0